### DISCUSSION TITLE
Error: Do not pass children as props.

### DISCUSSION THREAD

**Discussion by nyctonio** [remarkjs/remark#1100] [Q&A] [answered]
I got an error while working with NextJs 13 that in React Markdown i cant pass props as children how to get rid of this error
here is the code 

```
<ReactMarkdown remarkPlugins={[gfm]} className='prose' children={content} />
```

the problem is with the childeren={content}
how to solve this problem

**Comment by ChristianMurphy** [comment, marked as answer]
Without the actual error you're seeing it's hard to offer much support.

Taking an educated guess.
This likely isn't an actual error, as `children` is a valid prop.
It is most likely an ESLint rule warning:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-children-prop.md

Which you can disable/ignore if you want.
Or you could resolve by using
```jsx
<ReactMarkdown remarkPlugins={[gfm]} className='prose'>{content}</ReactMarkdown>
```
